core/src/test/java/kafka/test/junit/RaftClusterInvocationContext.java

```diff
@@ -284,24 +258,44 @@ public void startBroker(int brokerId) {
         @Override
         public void waitForReadyBrokers() throws InterruptedException {
             try {
-                clusterReference.get().waitForReadyBrokers();
+                clusterTestKit.waitForReadyBrokers();
             } catch (ExecutionException e) {
                 throw new AssertionError("Failed while waiting for brokers to become ready", e);
             }
         }
 
-        private BrokerServer findBrokerOrThrow(int brokerId) {
-            return Optional.ofNullable(clusterReference.get().brokers().get(brokerId))
-                .orElseThrow(() -> new IllegalArgumentException("Unknown brokerId " + brokerId));
-        }
-
         public Stream<BrokerServer> brokers() {
-            return clusterReference.get().brokers().values().stream();
+            return clusterTestKit.brokers().values().stream();
         }
 
         public Stream<ControllerServer> controllers() {
-            return clusterReference.get().controllers().values().stream();
+            return clusterTestKit.controllers().values().stream();
         }
 
+        private BrokerServer findBrokerOrThrow(int brokerId) {
+            return Optional.ofNullable(clusterTestKit.brokers().get(brokerId))
+                    .orElseThrow(() -> new IllegalArgumentException("Unknown brokerId " + brokerId));
+        }
+
+        private void buildAndFormatCluster() throws Exception {
```

Comment:
Could we add a phase `format` for kraft type? There are some test cases requiring `format`